tests/__init__.py

```diff
@@ -31,3 +34,11 @@
 
 if not os.path.isdir(_TEMP_PATH):
     os.mkdir(_TEMP_PATH)
+
+_SKIPIF_ARGS_PT_LE_1_4 = dict(condition=_TORCH_LOWER_EQUAL_1_4, reason="test pytorch > 1.4")
```

Comment:
Can you move all these unrelated test changes out of this PR? it will be easier to land

Reply:
It's just a lot of changes which we want to properly do, so better do it separate and not pollute the quantization PR

Reply:
I have a feeling that when I told you similar on your PR you didn't listen... :P
and this is related to this addition as you can see that we need to filer tests for PT <= 1.4

Reply:
Not the same because in my PR all changes were for pruning... Here there are changes to many other files